report/assets/styles.css

```
@@ -561,6 +562,38 @@
   border-bottom: var(--score-icon-size) solid var(--color-fail);
 }
 
+.lh-audit--error .lh-audit__score-icon,
+.lh-metric--error .lh-metric__icon {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  width: calc(var(--score-icon-size) * 2.4);
+  height: calc(var(--score-icon-size) * 2.4);
+  transform: translateX(-6px);
+  margin-right: 0;
+  border: none;
+}
+
+.lh-gauge__wrapper--fail .lh-gauge--error {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  left: 36px;
+  top: 8px;
+}
+
+.lh-sticky-header .lh-gauge__wrapper--fail .lh-gauge--error {
+  height: calc(var(--score-icon-size) * 3.25);
+  left: 10px;
+  top: 4px;
+}
+
+.lh-gauge__wrapper--fail:not([href]) .lh-gauge--error {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  height: 120px;
+  left: 139px;
+  top: -5px;
+}
```

We can avoid using exact pixel values with `background-position: center` and `var(--score-container-padding)`

```suggestion
.lh-gauge__wrapper--fail .lh-gauge--error {
  background-image: var(--error-icon-url);
  background-repeat: no-repeat;
  background-position: center;
  top: var(--score-container-padding);
}
```

---

Good callout, will update.